contents.json

```diff
@@ -27350,6 +27350,28 @@
       ],
       "date_added": "Dec 4 2021",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "Scribe - Language Keyboards",
+      "description": "Communicate with confidence using keyboards designed for language learners.”,
```

Comment:
Can you replace the `”` (smart quote?) at the end, should be `"`

Reply:
Was wondering about the red readout, thanks!